src/test/resources/model.yaml

```diff
@@ -28,32 +28,30 @@ defininitions:
     description: An employee of a company
     fields:
       id:
-        fieldDefinition: UUID
+        definition: UUID
       firstName:
-        fieldDefinition: Name
+        definition: Name
       lastName:
-        fieldDefinition: Name
+        definition: Name
       homeNumber:
-        fieldDefinition: PhoneNumber #assumes PhoneNumber but you can leave off #/fields/
```

Comment:
...assumes PhoneNumber but can be blank...  _maybe?_

_I know, it's cherry-picking but we try to avoid 1st and 2nd voice in tech docs_